README.md

```diff
@@ -7,25 +7,168 @@ An engineer working for a [CSE](./CSE.md) project...
 * Leads by example. Models the behaviors we desire both interpersonally and technically (this is what most people think of as a "developer").
 * Strives to understand how their work fits into a broader context and ensures the outcome.
 
-This is the playbook. You are invited to contribute via standard git pull requests to GitHub.
-
-## The Basics
-
-* [Source Control](./Engineering/SourceControl.md)
-* [Unit Testing](./Engineering/UnitTesting.md)
-* [Continuous Integration / Continuous Delivery](./Engineering/CICD.md)
-* [Code Reviews](./Engineering/CodeReviews.md)
-* [Retrospectives](./Engineering/Retrospectives.md)
-* [Engineering Practices And Principles](./Engineering/PracticiesAndPrinciples.md)
-
-## Seeking Contributions for
-
-- Component Tests
-- DevOps
-    - [Component Versioning](./Engineering/ComponentVersioning.md)
-    - [Logging and Monitoring](./Engineering/DevOpsLogging.md)
-    - Performance
-    - Telemetry
-- Security
-    - Threat Model
-- Regulatory Compliance
+This is the playbook. You are invited to contribute by submitting a pull request.
+
+## Outline
+
+The following outlines each section and what content would be included within each section.
+
+### Goals
+
+The proposed outline is intended to accomplish the following goals
+
+1. Author as idea-to-delivery value stream narrative from an engineering perspective.
+2. Organize for quick reference and discoverability.
+3. Extensible heirarchy to allow teams to share deep subject matter expertise.
+
+### Working Heirarchy
```

Comment:
Change to "A good Practice Working Hierarchy" as there will be variations. Also don't have the full outline in the READ.me but just add a link. Then move the whole section to a separate md file. This will keep the ReadMe clean as it only consists of the list of our 5 EF basics and a link to the a "A good Practice Working Hierarchy"

Reply:
We went with the term "Agile Lifecycle" to describe this flow instead of "Working Hierarchy". Note that the description mentions that this is *our* Agile Lifecycle (not necessarily *the*).

Reply:
Again i think the word Hierarchy is creating confusion on intent. The intent behind that language was to convey the designed folder structure for this repo, not the sprint cycle and team hierarchy. Looks like we need to swap a different word.